<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>826</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GayTor.rent Upload Utility</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/guu.png</normaloff>../icon/guu.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>627</width>
     <height>729</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="guu" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="fileBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>File</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="opaqueResize">
             <bool>false</bool>
            </property>
            <property name="childrenCollapsible">
             <bool>false</bool>
            </property>
            <widget class="QLabel" name="pathLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Path:</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="path"/>
            <widget class="QPushButton" name="fldrSelBtn">
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Select Folder</string>
             </property>
            </widget>
            <widget class="QPushButton" name="fileSelBtn">
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Select File</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="categoriesBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>165</height>
          </size>
         </property>
         <property name="title">
          <string>Categories</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="catWidget1" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="catLabel">
               <property name="minimumSize">
                <size>
                 <width>65</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>65</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>Category:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="category">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="catWidget2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="sc1Label">
               <property name="minimumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>Subcategory 1:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="subcategory1">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="sc3Label">
               <property name="minimumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>Subcategory 3:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="subcategory3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="sc2Label">
               <property name="minimumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>Subcategory 2:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="subcategory2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="sc4Label">
               <property name="minimumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>97</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>Subcategory 4:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="subcategory4">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="picturesBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>144</height>
          </size>
         </property>
         <property name="title">
          <string>Pictures</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QListWidget" name="picTable">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="selectionRectVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="picWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>103</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="addPicBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>60</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>2</width>
                 <height>1</height>
                </size>
               </property>
               <property name="text">
                <string>Add Picture(s)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="rmPicBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>45</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>1</width>
                 <height>1</height>
                </size>
               </property>
               <property name="text">
                <string>Remove Selected</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="infoBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>210</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Info</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="2" column="0" alignment="Qt::AlignRight">
              <widget class="QLabel" name="descLabel">
               <property name="text">
                <string>Description:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPlainTextEdit" name="description">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="title">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" alignment="Qt::AlignRight">
              <widget class="QLabel" name="titleLabel">
               <property name="text">
                <string>Title:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QProgressBar" name="uploadStatus">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>7</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="format">
                <string>Waiting... (%p%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="uploadBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Upload</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="statusBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>89</height>
          </size>
         </property>
         <property name="title">
          <string>Status</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="statusLabel1">
               <property name="text">
                <string>Torrent client:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="statusLabel2">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>error</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="statusLabel3">
               <property name="text">
                <string>Gaytor.rent:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="statusLabel4">
               <property name="text">
                <string>error</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="statusLabel5">
               <property name="text">
                <string>User:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="statusLabel6">
               <property name="text">
                <string>error</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="loginBtn">
               <property name="text">
                <string>Log In</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen_WebUI"/>
    <addaction name="actionReload_categories"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_WebUI">
   <property name="text">
    <string>Open client WebUI</string>
   </property>
  </action>
  <action name="actionReload_categories">
   <property name="text">
    <string>Reload categories</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
